src/tools/annotation/LengthTool.js

```diff
@@ -269,10 +287,7 @@ export default class LengthTool extends BaseAnnotationTool {
 
       annotation.unit = suffix;
 
-      return (
-        toGermanNumberStringTemp(measuredValue) +
-        ` ${suffix}` /*`${measuredValue.toFixed(2)} ${suffix}`*/
-      );
+      return `${measuredValue} ${suffix} +/- ${annotation.uncertainty}`;
```

Comment:
Where is measuredValue getting formatted?

Reply:
Have the questions about formatting been answered? Haven't we agreed that the formatting is going to be implemented as part of cornerstone localization PR?